scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/TimeRangeExpression.java

```diff
@@ -90,6 +90,7 @@ public Windows computeRange(Plan plan, Windows domain) {
       }
     }
     //out.println(name + " -> " + inter);
+    // if you uncomment the above line please use slf4j instead of println
```

Comment:
🔻 Rather than use our codebase as a message board, let's go ahead and delete commented out code 🤣 

For your context: the scheduler is one of the newest parts of the aerie system, and it's in the (painful) process of transforming from a prototype to a product. Hence the unused libraries, commented out code, etc.

Reply:
Whoops, yeah I realized that was dumb halfway through and removed most of them, must have forgot this one.